vizro-core/examples/visual-vocabulary/pages/examples/radar.py

```diff
@@ -0,0 +1,15 @@
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+
+wind = px.data.wind()
+
+page = vm.Page(
+    title="Radar chart",
+    components=[
+        vm.Graph(figure=px.line_polar(wind, r="frequency", theta="direction", color="strength", line_close=True))
+    ],
+)
+
+dashboard = vm.Dashboard(pages=[page])
+Vizro().build(dashboard).run()
```

Comment:
This example was already beautiful! I've made a few minor tweaks, mainly to simplify the chart. While the `color `dimension offers an interesting use-case, I recommend starting with just one category to keep the introduction of the chart straightforward.

I've also included a custom chart below to create the classic "radar" chart with a filled area. This custom chart is necessary so that we can utilize a post-update call - see our docs: https://vizro.readthedocs.io/en/stable/pages/user-guides/custom-charts/

Simply add the custom chart code to `custom_charts.py` and add some docstrings (refer to the butterfly example for guidance).

```suggestion
import vizro.models as vm
import vizro.plotly.express as px
from vizro import Vizro
from vizro.models.types import capture

wind = px.data.wind()


@capture("graph")
def radar(data_frame, **kwargs):
    """
    Add docstrings here (see butterfly example)
    
    """
    fig = px.line_polar(data_frame, **kwargs)
    fig.update_traces(fill="toself")
    return fig


page = vm.Page(
    title="Radar chart",
    components=[
        vm.Graph(
            figure=radar_chart(
                wind.query("strength =='1-2'"), r="frequency", theta="direction", color="strength", line_close=True
            )
        )
    ],
)

dashboard = vm.Dashboard(pages=[page])
Vizro().build(dashboard).run()
```